data/ingenic-6161.tablet

```diff
@@ -0,0 +1,18 @@
+# Ingenic Touch/Pen Display
+
+[Device]
+Name=Ingenic Touch/Pen display
+ModelName=Gadget_Serial_and_keyboard
```

Comment:
```suggestion
ModelName=
```
The model name is for cases where there's a marketing name shared across multiple models (Intuos Pro) but each one has a specific model name too. I think in this case we can just skip this, this looks like a template name that doesn't really specify this particular model.

Reply:
Change made in branch.

Reply:
thanks, I'll still need the sysinfo link please

Reply:
Uploaded here: https://github.com/linuxwacom/wacom-hid-descriptors/issues/346